OktaLoggerAppCenter.podspec

```diff
@@ -13,8 +13,9 @@ Pod::Spec.new do |s|
   s.static_framework = true
   
   s.source_files = [
-    'Sources/OktaLogger/AppCenterLogger/*'
+    'Sources/OktaLogger/AppCenterLogger/*',
+    'Sources/OktaLogger/LoggerCore/*.{h,m,swift}'
```

Comment:
Why?

Reply:
If I won't include it started failing on pod lib lint with an error: cannot find dependencies

Reply:
It worked fine before. What changed? Please investigate further. You don't need to include source code from dependency directly if you use s.dependency 'OktaLogger/Core','~>1.3'

Reply:
right, I am looking into it 
this is the error
```swift
    - ERROR | xcodebuild:  /Users/sandeeppenchala/okta/okta-logger-swift/Sources/OktaLogger/AppCenterLogger/AppCenterLogger.swift:19:35: error: cannot find type 'OktaLoggerLogLevel' in scope
    - ERROR | xcodebuild:  /Users/sandeeppenchala/okta/okta-logger-swift/Sources/OktaLogger/AppCenterLogger/AppCenterLogger.swift:19:24: error: method does not override any method from its superclass
    - ERROR | xcodebuild:  /Users/sandeeppenchala/okta/okta-logger-swift/Sources/OktaLogger/AppCenterLogger/AppCenterLogger.swift:18:24: error: cannot find type 'OktaLoggerDestinationBase' in scope
```

Reply:
Hmm, does it mean `OktaLoggerLogLevel` and `OktaLoggerDestinationBase` are not part of Core SDK? We need to investigate further

Reply:
both are available in another module public and open

Reply:
Ok, then I suppose you have to import OktaLogger in the source code in order to use them...